<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Sebastian Riedel</author>
 <class>NewMap</class>
 <widget class="QDialog" name="NewMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New Map</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/mk/shared/editor.ico</normaloff>:/mk/shared/editor.ico</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelWidth">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHeight">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelSurface">
       <property name="text">
        <string>Surface</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAltitude">
       <property name="text">
        <string>Altitude</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPlayers">
       <property name="text">
        <string>Number of players</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="inputPlayers">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="inputSurface">
       <property name="toolTip">
        <string>Default surface material</string>
       </property>
       <item>
        <property name="text">
         <string>1 - Grass</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_grass1.png</normaloff>:/icons/brush_surface_grass1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 - Secondary grass</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_grass2.png</normaloff>:/icons/brush_surface_grass2.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 - Road</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_road.png</normaloff>:/icons/brush_surface_road.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 - Stone</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_stone.png</normaloff>:/icons/brush_surface_stone.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 - Ground</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_custom.png</normaloff>:/icons/brush_surface_custom.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="inputWidth">
       <property name="toolTip">
        <string>must be 16, 32, 64, 128, 256, 512 …</string>
       </property>
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="inputHeight">
       <property name="toolTip">
        <string>must be 16, 32, 64, 128, 256, 512 …</string>
       </property>
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="inputAltitude">
       <property name="toolTip">
        <string>Default surface height</string>
       </property>
       <property name="maximum">
        <number>40</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputSurface</tabstop>
  <tabstop>inputPlayers</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewMap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewMap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
